compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposeLayer.desktop.kt

```diff
@@ -331,17 +331,14 @@ internal class ComposeLayer(
         _component.addInputMethodListener(object : InputMethodListener {
             override fun caretPositionChanged(event: InputMethodEvent?) {
                 if (isDisposed) return
-                if (event != null) {
-                    catchExceptions {
-                        platform.textInputService.onInputEvent(event)
-                    }
-                }
+                // Which OSes and which input method could produce such events? We need to have some
```

Comment:
Maybe we should ask someone knowledgeable? I bet it's not here for nothing.

Reply:
It [may be](https://stackoverflow.com/questions/19755833/caretpositionchanged-method-not-reacting) only for Swing components. This method isn't triggered on Linux/Windows/macOS

I created a separate issue that can be related to this: https://github.com/JetBrains/compose-jb/issues/2758